packages/xkbcomp.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Xkbcomp < Package
+  description 'Compile XKB keyboard'
+  homepage 'https://www.x.org'
+  version '1.4.1'
+  source_url 'https://www.x.org/archive//individual/app/xkbcomp-1.4.1.tar.gz'
+  source_sha256 '9cdae907b6e34f1f325f6311020126157037925877f323a880e0c9ecab9628f6'
+
+
+  depends_on 'mesa'
+  depends_on 'xcb_util'
+
+    
+  def self.build
+    system "./configure \
+            --prefix=#{CREW_PREFIX} \
+            --libdir=#{CREW_LIB_PREFIX} \ 
+            --with-xkb-config-root=#{CREW_LIB_PREFIX}/X11/xkb "
+    system "make"
+  end
+
+  def self.install
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+  end
+  
+  def self.postinstall
+    system "mkdir -p #{CREW_LIB_PREFIX}/X11/xkb"
```

Comment:
Actually, I meant this line can stay in the `def self.install` section but if you needed any messages about the configuration that would be needed to populate this directory, those can go in the `def self.postinstall` section.  Sorry about the confusion.  Just so you know, the `def self.postinstall` section lines are executed regardless of whether you install via source or binary, however, the `def self.install` section is only executed during the source install.  Hopefully, that makes sense.